app/lib/concerns/versioning_helpers.rb

```diff
@@ -0,0 +1,5 @@
+module VersioningHelpers
+  def minor_version(version)
+    (version.scan(/1\.(\d+)/).flatten.first.to_i || 0)
```

Comment:
Rather than parsing ourselves, could we use Stdlib? `Gem::Version.new("1.0.0").segments` https://metaredux.com/posts/2018/10/28/a-better-way-to-compare-versions-in-ruby.html

Reply:
That looks a lot better, will update.